tools/fuse-wake/fuse-wake.cpp

```diff
@@ -34,20 +34,20 @@
 
 int main(int argc, char *argv[])
 {
-	if (argc != 3) {
-		std::cerr << "Syntax: fuse-wake <input-json> <output-json>" << std::endl;
+	if (argc != 5 || 0 != strncmp(argv[1], "-p", 2) || 0 != strncmp(argv[3], "-o", 2)) {
```

Comment:
Why do we need to add these markers? Just for clarity on the command line?

Reply:
yeah otherwise someone might write `wakebox -o outputfile -p inputfile` and they'd be suprised that inputfile was written to